addons/binding/org.openhab.binding.lgwebos/META-INF/MANIFEST.MF

```diff
@@ -22,6 +22,7 @@ Import-Package:
  org.apache.http.message,
  org.apache.http.protocol,
  org.eclipse.jdt.annotation;resolution:=optional,
+ org.eclipse.smarthome.automation.annotation,
```

Comment:
you should make this an optional import, so that the binding does not depend on the presence of the new rule engine.

Reply:
Done